Tasks/AzureKeyVaultV2/operations/azure-arm-keyvault.ts

```diff
@@ -59,7 +59,8 @@ export class KeyVaultClient extends azureServiceClient.ServiceClient {
 
                 return response;
             } catch(error) {
-                if (retriableErrorCodes.indexOf(error.code) != -1 && ++retryCount < maxRetryCount) {
```

Comment:
Seems like the same logic is in the V1 version of the task. Does it make sense to change it there too?

Reply:
There is this issue related to V1 https://github.com/microsoft/azure-pipelines-tasks/issues/15052. Maybe it has the same root cause.

Reply:
Ok, I will add the same change for V1 as well.
About the issue: it has another error message so it won't be fixed by the current change, more details about the error for it can be found in https://www.codingdefined.com/2015/06/nodejs-error-errno-eaiagain.html